app/src/main/java/com/kevo/displaydemo/service/AccessoryDisplayService.kt

```diff
@@ -0,0 +1,126 @@
+package com.kevo.displaydemo.service
+
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.Service
+import android.content.Intent
+import android.os.Binder
+import android.os.Build
+import android.os.IBinder
+import android.util.Log
+import androidx.annotation.DrawableRes
+import androidx.core.app.NotificationCompat
+import com.kevo.displaydemo.R
+
+class AccessoryDisplayService : Service() {
+
+    private var isForeground = false
+    private var isBound = false
+
+    /**
+     * [Binder] for the [AccessoryDisplayService].
+     */
+    inner class ServiceBinder : Binder() {
+
+        var service: AccessoryDisplayService = this@AccessoryDisplayService
+    }
+
+    override fun onCreate() {
+        super.onCreate()
+        Log.d(TAG, "onCreate -> Service is being created ...")
+
+        isForeground = true
+        startForeground(REGISTER_ACCESSORY_DISPLAY_SERVICE, getNotification())
+    }
+
+    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
+        Log.d(TAG, "onStartCommand -> Command configuration: $intent / $flags / $startId")
+        return START_STICKY
+    }
+
+    override fun onBind(intent: Intent): IBinder {
+        Log.d(TAG, "onBind -> Service bound using $intent")
+        return ServiceBinder()
+    }
```

Comment:
We should set `isBound = true` in this `onBind` callback.

Reply:
Done 👍 Service infrastructure to be refined in subsequent commits